Example/Example/LottieViewLayoutDemoView.swift

```diff
@@ -9,10 +9,11 @@ struct LottieViewLayoutDemoView: View {
     HStack {
       VStack {
         LottieView(animation: .named("Samples/LottieLogo1"))
+          .configure(\.contentMode, value: .scaleAspectFit)
```

Comment:
I was also wondering if it would be nicer to omit the labels entirely:

```suggestion
          .configure(\.contentMode, .scaleAspectFit)
```

If you think that's nicer I'm happy to adjust.

Reply:
Yup, I like it!